scripts/update-requirements.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/bash
```

Comment:
Maybe I am wrong, but what is purpose of this script? Upgrading of deps versions in req files and eventually reverting it? I can do it by two commands:
~~~bash
pur -r requirements.txt  # upgrade
git checkout . # revert
~~~
We do not need another helper bash script for that.

Reply:
If it would be documented somewhere (confluence?) how to do the upgrade of libs and pur added to the requirements.txt (because it's dev dependency) then maybe we don't need the script

Reply:
Maybe we could add this line some into CI pipelines.
~~~bash
pur -r requirements.txt -o /dev/null | grep "Updated" | sed -e "s/Updated/You can update:/g"
# Output:
You can update: pytest: 3.6 -> 5.0.1
You can update: aiokafka: 0.5.0 -> 0.5.2
You can update: certifi: 2019.3.9 -> 2019.6.16
....
~~~
It will automatically inform as about updates.